### ISSUE TITLE
Each Korean name is displayed in the wrong order

### ISSUE THREAD

**Issue by mndscp** [SimpleMobileTools/Simple-Contacts#873]
Hey, thanks for all the simple apps, priceless! :)

Korean names are displayed the wrong way.
They usually consist of three parts: 1 character surname, 2 characters first name, and they are written without spaces in that order.
Example: 박민주 with 박 as surname and 민수 as first name

However, in the app they are being turned around just like non-Asian names I guess.
박민수 becomes 민수 박 in the app, even with a space in between. This looks quite wrong, it's not an American or European name.
I tried to work around this by not filling out the surname and only filling out the first name with the full name in the correct order 박민수 but it did not help.

TL;DR: Korean full names are displayed in a weird and wrong way like 민수 박 with 박 as surname and 민수 as first name.
They should be displayed like <surname><first name> without a space in between like 박민수.

I hope the description helps. Let me know if you need more info.
Have a nice day!
